packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -616,6 +616,39 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		this[__private__dont__use].FS.rename(fromPath, toPath);
 	}
 
+	/** @inheritDoc */
+	@rethrowFileSystemError('Could not copy "{path}"')
+	cp(fromPath: string, toPath: string) {
+		const FS = this[__private__dont__use].FS;
+
+		const fromStat = FS.stat(fromPath);
+
+		const fromMode = parseInt(fromStat.mode.toString(8).substring(0, 1));
+
+		const fromIsDir = fromMode === 4;
```

Comment:
ditto

Reply:
https://github.com/WordPress/wordpress-playground/pull/846/files#diff-b33e1e6834fb94ac2cb5338e3ae17c60abf7ac88cb057189a1915531e55f7243R629

Reply:
@seanmorris as I mentioned in another PR, links pointing to the `/files` tab become stale once the HEAD pf this branch changes. To share links that remain valid as the branch changes, you'll have to link to changes in specific commits.

Reply:
> @seanmorris thanks! As a side note – as I mentioned in another PR, links pointing to the `/files` tab become stale once the HEAD pf this branch changes. To share links that remain valid as the branch changes, you'll have to link to changes in specific commits.

I was wondering how you were formatting that code right in the comments. Thanks for showing me that.